code/modules/mob/living/carbon/carbon.dm

```diff
@@ -215,8 +210,8 @@
 			AdjustKnockdown(-3)
 			playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 			M.visible_message( \
-				"<span class='notice'>[M] shakes [src] trying to wake [t_him] up!</span>", \
-				"<span class='notice'>You shake [src] trying to wake [t_him] up!</span>", \
+				"<span class='notice'>[M] shakes [src] trying to wake \him up!</span>", \
```

Comment:
>Not in the repo's history, which is the main one that matters. 

>Commit to change carbon.dm for this file
>Commit to change carbon.dm for the other PR for the sake of atomicity

???

Regardless this is tested, works, and is part of a larger cleanup of the file.